src/main/java/org/xstefank/model/yaml/Format.java

```diff
@@ -16,19 +17,15 @@ public void setTitle(String title) {
         this.title = title;
     }
 
-    public Description getDescription() {
-        return description;
-    }
+    public String getCommit() { return commit; }
```

Comment:
I don't have checkstyle set up (that's on me), but please do not use one line methods